### DISCUSSION TITLE
Next Version Release 2.0.0

### DISCUSSION THREAD

**Discussion by leifermendez** [codigoencasa/builderbot#105] [Ideas]
__Package CLI__
> La idea de tener un CLI (Command Line Interface) es poder tener una herramienta que sea capaz de gestión algunos de los siguientes problemas comunes

__RoadMap:__ [https://github.com/users/leifermendez/projects/4](https://github.com/users/leifermendez/projects/4)
Pueden estar pendiente a las nuevas features

- Que actualice y corrija los problema de versionamientos más frecuetes
-  Verificar la versión de NODE correcta
- Verificar OS para brindar mejor soporte de puppeter
-  Limpiar sesion, borrar carpeta de sesion
- Seleccionar provider
- Poder usar un archivo .json con la configuración 

__Package Provider__
> Es parte clave del proyecto la idea es poder tener la opcion de incluir otro proveedor de mensajeria como la api oficial o api de twilio

- WhatsappWeb (default)
- Whatsapp API official
- Twilio

__Package Input/Output__
> Gestionar los diferentes mensajes entranates y poder responder, a la vez de mantener un registro de los datos

**Comment by leifermendez** [comment]
Ya se lleva avanzado el 70% de todas estas features 💪

**Comment by sotile** [comment]
podria implementarse una api para enviar mensajes sin que el cliente escriba primero? conectar a una base de datos para recordarle una cita agendada al cliente o similar?

**Reply by jfrancisco2803** [reply]
se podria implementar para que el mismo bot envie un mensaje de confirmacion a un correo yde que pueda leer a partir de un excel como base de datos
